homeassistant/components/homekit_controller.py

```diff
@@ -0,0 +1,178 @@
+"""
+Support for Homekit device discovery.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/homekit_controller/
+"""
+import http
+import json
+import logging
+import os
+import uuid
+
+from homeassistant.components.discovery import SERVICE_HOMEKIT
+from homeassistant.config import get_default_config_dir
+from homeassistant.helpers import discovery
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['homekit==0.5']
+
+homekit_dir = 'homekit'
+DOMAIN = 'homekit_controller'
+
+_CONFIGURING = {}
+
+# Mapping from Homekit type to component.
+HOMEKIT_ACCESSORY_DISPATCH = {
+    'lightbulb': 'light',
+    'outlet': 'switch',
+}
+
+KNOWN_ACCESSORIES = []
```

Comment:
These should also be moved to `hass.data`. Use any key you want in the dict but since it is shared among all components, prefix your key with your domain.

Reply:
Done